scripts/linters/pre_commit_linter.py

```diff
@@ -461,13 +465,30 @@ def main(args=None):
     # File extension to be linted.
     file_extension_types = _get_file_extensions(
         parsed_args.only_check_file_extensions)
+
+    # These environmental variables are required to allow Google Cloud Tasks to
+    # operate in a local development environment without connecting to the
+    # internet. These environment variables allow Cloud APIs to be instantiated.
+    os.environ['CLOUDSDK_CORE_PROJECT'] = 'oppia-dev'
```

Comment:
Call it dummy-cloudsdk-project-name or similar, ditto below -- make clear what its purpose is just from reading the name, if it shows up in the logs when an error occurs.